modules/ROOT/pages/auto-upgrade-runtime-version-used.adoc

```diff
@@ -0,0 +1,12 @@
+= Automatic Upgrades of Mule-App Projects to the Latest Mule Runtime
+
+After a new version of Mule runtime is released, opening an existing Mule-app project causes the application to use that new version. Immediately the application is upgraded, Design Center restarts it in the Design environment only. If Design Center is unable to upgrade an application, then the application is not restarted.
```

Comment:
I suggest: "The application is upgraded immediately and Design Center restarts it in the Design environment only." It sounds funny to me to start with "Immediately."

Reply:
Thanks. I'll make the changes eventually. I forgot that this file isn't supposed to be published for this release; so, I've removed it.